trainer/trainer_utils.py

```diff
@@ -98,7 +98,7 @@ def setup_torch_training_env(
         num_gpus = torch.cuda.device_count()
 
     if num_gpus > 1 and (not use_ddp and not args.use_accelerate):
-        msg = f" [!] {num_gpus} active GPUs. Define the target GPU by `CUDA_VISIBLE_DEVICES`. For multi-gpu training use `TTS/bin/distribute.py`."
+        msg = f" [!] {num_gpus} active GPUs. Define the target GPU by `CUDA_VISIBLE_DEVICES`. For multi-gpu training use `trainer/distribute.py`."
```

Comment:
Actually this is better because most users wouldn't have the Trainer repo cloned:
```suggestion
        msg = f" [!] {num_gpus} active GPUs. Define the target GPU by `CUDA_VISIBLE_DEVICES`. For multi-gpu training use `python -m trainer.distribute`."
```